### ISSUE TITLE
[SF][A11y][Breadcrumbs]: In breadcrumb trail, current page element is announced as link but element cannot be activated

### ISSUE THREAD

**Issue by VidhiNPatel** [UI5/webcomponents#12780]
### Describe the bug

In the breadcrumb trail, the current page element is indicated as a link, but it cannot be activated. The role="link" attribute should be removed from the current page element to align with the UI5 page behavior.

### Reproduction steps

1. Go to https://ui5.github.io/webcomponents-react/v2/?path=/docs/layouts-floorplans-breadcrumbs--docs 
2. Inspect the Example 
3. Observe that the "Notebooks" text for the current page has a role="link"

### Expected Behaviour

The role="link" attribute should be removed from the current page element to align with the UI5 page behavior.

Reference - https://ui5.sap.com/#/entity/sap.m.Breadcrumbs/sample/sap.m.sample.Breadcrumbs

### UI5 Web Components for React Version

2.16.0

### UI5 Web Components Version

2.16

### Browser

Chrome

### Relevant log output

```shell

```

### Declaration

- [x] I’m not disclosing any internal or sensitive information.

**Comment by MarcusNotheis** [comment]
Thanks for reporting! I'll forward this issue to our UI5 Web Components Colleagues as the affected component is developed in their repository.

**Comment by didip1000** [comment]
Hey @UI5/ui5-team-ctr-pir,

Can you take a look at this issue? 

Best,
Diana
